src/components/CalculatorInput.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import '../styles/CalculatorInput.css';
+
+const CalculatorInput = ({ number }) => (
+  <div className="calculator-input">
+    <span>{number}</span>
```

Comment:
- [ ] Well done all the UI looks similar to the original design, just a detail, the default number on the display is 0, please 🙏🏽 replace it to be 100% similar to the original design.